packages/gatsby-plugin-sharp/README.md

```diff
@@ -30,9 +30,13 @@ plugins: [
   {
     resolve: `gatsby-plugin-sharp`,
     options: {
-      useMozJpeg: false,
+      // Available options and their defaults:
+      base64Width: 20,
+      forceBase64Format: ``, // valid formats: png,jpg,webp
+      useMozJpeg: process.env.GATSBY_JPEG_ENCODER === `MOZJPEG`,
       stripMetadata: true,
-      defaultQuality: 75,
+      lazyImageGeneration: true,
```

Comment:
Currently, this argument doesn't do anything. So we shouldn't display it in my opinion.
```suggestion
```
```